### ISSUE TITLE
Blue line appearing at single zoom level

### ISSUE THREAD

**Issue by mjcunningham** [tangrams/eraser-map#183]
This is in the prod build. See attached. 
![screenshot_20160608-091047](https://cloud.githubusercontent.com/assets/307758/15895612/27adcb4a-2d5a-11e6-8390-e8f0583ef7d7.png)

**Comment by matteblair** [comment]
Probably fixed by https://github.com/tangrams/bubble-wrap/commit/1746d430529a1f85393d3f2c56b5286e06422f90

We should update the versions of bubble-wrap in the Android SDK and Eraser Map

**Comment by nvkelso** [comment]
Yes, fixed by https://github.com/tangrams/bubble-wrap/commit/1746d430529a1f85393d3f2c56b5286e06422f90.

I've released [v2.0.1](https://github.com/tangrams/bubble-wrap/releases/tag/v2.0.1), please update Eraser Map to point at this new release.

**Comment by matteblair** [comment]
Is there a similar issue in Cinnabar or Refill? If so we can update those as well.

**Comment by matteblair** [comment]
Oh actually it looks like this fix is already in the submodule commit referenced by the Android SDK ([according to this history](https://github.com/tangrams/bubble-wrap/commits/1e100fbc0a9364de188ad35738853a7c57dc1f58)). Since Eraser Map uses styles from the Android SDK now, the next release will have this fix.

**Comment by ecgreb** [comment]
Once we wrap up the current round of SDK changes we'll push a new version of the app to the store.
